managed/src/main/java/com/yugabyte/yw/models/CustomerConfig.java

```diff
@@ -135,6 +137,15 @@ public static CustomerConfig get(UUID customerUUID, UUID configUUID) {
       .findOne();
   }
 
+  public static CustomerConfig getOrBadRequest(UUID customerUUID, UUID configUUID) {
+    CustomerConfig storageConfig = get(customerUUID, configUUID);
+    if (storageConfig == null) {
+      throw new YWServiceException(BAD_REQUEST, "Invalid StorageConfig UUID: "
+          + configUUID);
+    }
+    return storageConfig;
+  }
+
```

Comment:
Mark get method @Deprecated so people are encouraged to use getOrBadRequest

Reply:
Done.